diff --git a/node_modules/react-native/sdks/hermes-engine/utils/replace_hermes_version.js b/node_modules/react-native/sdks/hermes-engine/utils/replace_hermes_version.js
index 3b63e97..3113218 100644
--- a/node_modules/react-native/sdks/hermes-engine/utils/replace_hermes_version.js
+++ b/node_modules/react-native/sdks/hermes-engine/utils/replace_hermes_version.js
@@ -10,7 +10,7 @@
 
 const yargs = require('yargs');
 const fs = require('fs');
-const {execSync} = require('child_process');
+const {execFileSync} = require('child_process');
 
 const LAST_BUILD_FILENAME = '.last_build_configuration';
 
@@ -61,7 +61,9 @@ function replaceHermesConfiguration(configuration, version, podsRoot) {
   fs.mkdirSync(finalLocation, {recursive: true});
 
   console.log('Extracting the tarball');
-  execSync(`tar -xf ${tarballURLPath} -C ${finalLocation}`);
+  execFileSync('tar', ['-xf', tarballURLPath, '-C', finalLocation], {
+    stdio: 'inherit',
+  });
 }
 
 function updateLastBuildConfiguration(configuration) {
